### DISCUSSION TITLE
Konga Beat – A custom track editor for Donkey Konga 2 and 3

### DISCUSSION THREAD

**Discussion by parhamgholami** [melanchall/drywetmidi#314] [Show and tell]
Back in the early 2000s, Nintendo and Namco developed a series of music rhythm games for the GameCube featuring Donkey Kong called Donkey Konga: https://www.mariowiki.com/Donkey_Konga_(series)

The Donkey Konga games borrowed heavily from Taiko no Tatsujin (another music rhythm game by Namco). However, instead of taiko drums, the player would use DK Bongos to jam along with music from different eras and genres.

Long story short, I figured out how to add custom tracks to some of the Donkey Konga games (Donkey Konga 2 and 3) but found the entire process cumbersome, so I decided to make a dedicated editor called [Konga Beat](https://www.kongabeat.com/). Since Donkey Konga 2 and 3 scores are managed as MIDI files, DryWetMidi played a very big role in this project. Thank you for building such a great library! 😄

**Comment by melanchall** [comment]
Hi @parhamgholami,

Thank you for using the library and good words about it! I'm glad you found the library useful. Good luck with your projects :-)
